src/js/options.js

```diff
@@ -133,7 +137,7 @@ Object.entries(SETTINGS_LIST).forEach(([key, { eventType }]) => {
     }
 
     // Update options page.
-    Object.entries(saveObj).forEach(([key, value]) => HTML.setAttribute(key, value));
+    Object.entries(saveObj).forEach(([key, value]) => HTML.setAttribute(key, values?.[value] ?? value));
```

Comment:
We need to set the value in `saveObj` to the key from `values` but the attribute to the corresponding value.